### PR TITLE
fix readme typo

### PR DESCRIPTION
**Pull request by fukusuket**
Fixed the following typos in README.

- pivot-keywords-listコマンド heading level
- rustc requirements version
- description of logon-summary
- description of command help(pivot-keywords-list/metrics)

I would appreciate it if you could review🙏

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```
Comment (hitenkoku) [inline .idea/.gitignore:1]: What is the reason for adding this file? It looks like a personal environment configuration file. If you don't need it, please delete it.

File: .idea/modules.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (hitenkoku) [inline .idea/modules.xml:1]: What is the reason for adding this file? It looks like a personal environment configuration file. If you don't need it, please delete it.

File: .idea/vcs.xml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (hitenkoku) [inline .idea/vcs.xml:1]: What is the reason for adding this file? It looks like a personal environment configuration file. If you don't need it, please delete it.

**Review by hitenkoku — CHANGES_REQUESTED** [review]
@fukusuket Thanks for your pull requst.
It looks added Some personal environment configuration file.
Would you check and answer?

**Comment by fukusuket** [comment]
@hitenkoku  Sorry🙇 it was my commit mistake. Thank you for quick review. I've deleted unneeded files.

File: .idea/.gitignore (hunk 1 above)
Comment (hitenkoku) [reply .idea/.gitignore]: fixed by https://github.com/Yamato-Security/hayabusa/pull/876/commits/b82087ba6bedfddc656f1faeb130583090bdbed7

File: .idea/modules.xml (hunk 2 above)
Comment (hitenkoku) [reply .idea/modules.xml]: fixed by https://github.com/Yamato-Security/hayabusa/pull/876/commits/b82087ba6bedfddc656f1faeb130583090bdbed7

File: .idea/vcs.xml (hunk 3 above)
Comment (hitenkoku) [reply .idea/vcs.xml]: fixed by https://github.com/Yamato-Security/hayabusa/pull/876/commits/b82087ba6bedfddc656f1faeb130583090bdbed7

**Review by hitenkoku — APPROVED** [review]
@fukusuket Thanks for the very quick fix!

LGTM

**Comment by hitenkoku** [comment]
Squashed Merge to prevent personal settings from entering main commits.

**Pull request merged**